packages/components/src/components/data-grid/cell-handlers/graph-cell.tsx

```diff
@@ -27,7 +27,11 @@ export const GraphCell: Cell = {
     const { style = 'progress', min = 0, max = 100 } = field;
 
     // Convert content to 0>100 range for progress bar
-    const progress = ((content - min) / (max - min)) * 100;
+    const progress = parseFloat(
+      (((content - min) / (max - min)) * 100).toPrecision(
+        String(content).replace('.', '').length
```

Comment:
I think we should hard code the decimal point value here, or give an optional prop for the user to decide how many decimals should be displayed.

String(content).replace('.', '').length -> this could lead to the same issue again if, for example the content is 19210.9999 it would still render a very long number

Reply:
<img width="1792" alt="Screenshot 2023-05-16 at 2 08 41 PM" src="https://github.com/telekom/scale/assets/126784820/f911d620-42a7-48ce-867f-e31d90c6adc4">
I tested with long number as well, using the toPrecision function it is working fine and it will take data as whatever user has provided in datagrid rows.

Reply:
this was my change to test it:
 
 <img width="609" alt="Screenshot 2023-05-16 at 11 06 34" src="https://github.com/telekom/scale/assets/97440128/f1b618c9-ebc4-4341-8e4f-de2f6c82a3d5">

 and this is the output:
 
 
<img width="1155" alt="Screenshot 2023-05-16 at 11 07 04" src="https://github.com/telekom/scale/assets/97440128/26775204-a710-4c00-b09c-7639842e4b9e">

Reply:
not sure if i understood something wrong but the i thought the issue was the long decimals in the progress bar precentage

Reply:
thanks for clarifying @ayeshaa-m, i understood the issue wrong